triagebot.toml

```
@@ -1424,3 +1424,6 @@ compiletest = [
 # Enable `@rustbot note` functionality
 # Documentation at: https://forge.rust-lang.org/triagebot/note.html
 [note]
+
+[behind-upstream]
+days-threshold = 7
```

@wesleywiser and I talked about it yesterday at RustWeek, and we would like to try at 14 days to start. We can adjust it later if it's too much.
```suggestion
days-threshold = 14
```

---

Make sense :)